data/src/test/java/io/homeassistant/companion/android/data/authentication/AuthenticationRepositoryImplSpec.kt

```diff
@@ -209,6 +232,26 @@ object AuthenticationRepositoryImplSpec : Spek({
                     assertThat(externalAuth).isEqualTo("{\"access_token\":\"HGFEDCBA\",\"expires_in\":1800}")
                 }
             }
+
+            describe("build bearer token") {
+                lateinit var token: String
+                beforeEachTest {
+                    token = runBlocking { repository.buildBearerToken() }
+                }
+                it("should refresh the token") {
```

Comment:
Just for information.
Usually I do not verify the dependency call if the function is returning something.
`assertThat(token).isEqualTo("Bearer HGFEDCBA")` is enough the value can only be return from the mockk